src/test/java/io/rcktapp/api/handler/sql/SqlGetHandlerTest.java

```diff
@@ -0,0 +1,123 @@
+package io.rcktapp.api.handler.sql;
+
+import io.forty11.j.J;
+import io.forty11.sql.Sql;
+import io.forty11.web.Url;
+import io.rcktapp.api.*;
+
+import io.rcktapp.api.Collection;
+import io.rcktapp.api.service.Service;
+
+import io.rcktapp.rql.Replacer;
+import io.rcktapp.rql.Rql;
+import io.rcktapp.rql.Stmt;
+import io.rcktapp.rql.sql.SqlRql;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.*;
+
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.sql.Connection;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class SqlGetHandlerTest {
+
+    @Mock
+    Service service;
+
+    @Mock
+    Api api;
+
+    @Mock
+    Endpoint endpoint;
+
+    @Mock
+    Action action;
+
+    @Mock
+    Chain chain;
+
+    @Mock
+    Response response;
+
+    @Mock
+    SqlDb sqlDb;
+
+    @Mock
+    SqlRql sqlRql;
+
+    @Mock
+    Stmt stmt;
+
+    @Mock
+    Connection connection;
+
+    @Mock
+    SqlGetHandler handler;
+
+    @Captor
+    ArgumentCaptor<List> queryParametersCaptor;
+
+    @Captor
+    ArgumentCaptor<String> sqlCaptor;
+
+    @Test
+    public void testSqlGetHandler_queryParametersAreProperlyPassedToQueryObjectsMethod() throws Exception {
+
+        Url url = new Url("http://localhost:8080/api/lift/us/plugins/updatexml(rand(),concat(CHAR(126),user(),CHAR(126)),null)");
+
+        String passedQuery = "updatexml(rand(),concat(CHAR(126),user(),CHAR(126)),null)";
+        String collectionName = "plugins";
+        String expectedQuery = " SELECT * FROM " + collectionName + " WHERE plugins IN (?,?,?,?,?) ";
+        String [] inClause = {"updatexml(rand()", "concat(CHAR(126)", "user()", "CHAR(126))", "null)"};
+
+        // Setup request object and sql table objects to properly reflect
+        Request request = new Request(url, "GET", new HashMap<>(), new HashMap<>(), "");
+        Service.ApiMatch apiMatch = new Service.ApiMatch(api, endpoint, "GET", request.getUrl(), "http://localhost:8080/api/lift/us/", collectionName + "/" + passedQuery + "/");
+        Collection collection = new Collection();
+        Entity entity = new Entity(new Table(sqlDb, collectionName));
+        Attribute attribute = new Attribute();
+
+        attribute.setName(collectionName);
+        entity.setKey(attribute);
+        collection.setEntity(entity);
+        request.setApiMatch(apiMatch);
+
+        mockStatic(Rql.class);
+        mockStatic(Sql.class);
```

Comment:
I think we would want to include these statements in a try-with-resources block. Otherwise, these RQL and SQL static classes would remained mocked outside the scope of this test, which could impact other tests that may use these in the future

Checkout the bolded sections on this article: https://www.baeldung.com/mockito-mock-static-methods#mocking-a-no-argument-static-method